webapi/src/main/scala/org/knora/webapi/messages/v1/responder/usermessages/UserMessagesV1.scala

```diff
@@ -442,21 +633,48 @@ object UserProfileType extends Enumeration {
     }
 }
 
+
+/**
+  * Payload used for updating of an existing user.
+  *
+  * @param email         the new email address. Needs to be unique on the server.
+  * @param givenName     the new given name.
+  * @param familyName    the new family name.
+  * @param password      the new password.
+  * @param status        the new status.
+  * @param lang          the new language.
+  * @param projects      the new project memberships list.
+  * @param projectsAdmin the new projects admin membership list.
+  * @param groups        the new group memberships list.
+  * @param systemAdmin   the new system admin membership
+  */
```

Comment:
I have added it to the construction. I'm not sure about the practicality in this case. Will see how it plays out over time.